install/amazon_rds/03_install_collector_eks.mdx

```diff
@@ -0,0 +1,59 @@
+---
+title: 'Step 3: Install the Collector'
+install_track_title: Installation Guide (Amazon RDS and Amazon Aurora)
+backlink_href: /docs/install
+backlink_title: 'Installation Guide'
+---
+
+import TabPanel, {TabTextContent} from '../../components/TabPanel'
+import SetupIAMPolicy from './_03_setup_iam_policy.mdx';
+import CreateIAMRoleEKSIRSA from './_03_create_iam_role_eks_irsa.mdx'
+import CreateIAMRoleEKSPodIdentity from './_03_create_iam_role_eks_pod_identity.mdx'
+
+export const IAMRoleCreationEKS = () => {
+  const tabs = [
+    [ 'irsa', 'IAM roles for service accounts (IRSA)' ],
+    [ 'pod_identity', 'Pod Identity'],
+  ];
+  return (
+    <TabPanel items={tabs}>
+      {(idx) => {
+        const tab = tabs[idx][0];
+        switch (tab) {
+          case 'irsa':
+            return <TabTextContent><CreateIAMRoleEKSIRSA /></TabTextContent>;
+          case 'pod_identity':
+            return (
+              <TabTextContent>
+                Creating an IAM role with Amazon EKS Pod Identity is not
+                supported yet. Please use IAM roles for service accounts (IRSA).
+              </TabTextContent>
```

Comment:
Yeah, +1 to doing this inline in the text to warn that it's not supported, instead of using tabs.

Also, is it actually not supported, or are we mainly missing documentation? (I haven't looked at Pod Identity much, so don't know if it needs changes on the collector code side to work?)